src/main/java/org/fundacionjala/coding/omar/movies/MovieNew.java

```diff
@@ -0,0 +1,37 @@
+package org.fundacionjala.coding.omar.movies;
+
+/**
+ * Created by Omar Limbert Huanca Sanchez AT06.
+ */
+public class MovieNew extends AbstractMovie {
+
+    private static final int NUMBER_3 = 3;
+    private static final int NUMBER_1 = 1;
+    private static final int NUMBER_2 = 2;
+
+    /**
+     * Constructor for MovieNew.
+     *
+     * @param title Title of new movie.
+     */
+    public MovieNew(String title) {
+        super(title);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    double getAmount(int daysRented) {
+        return daysRented * NUMBER_3;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)